README.md

```diff
@@ -449,11 +449,12 @@ docker run \
 The micro-ROS Agent can also be built as a ROS 2 package in a Colcon workspace.
 This procedure is rather involved, so only do this if the pre-configured Docker image can not be used.
 
-The following sections show how to build the Humble version of the Agent in a dedicated workspace (adapt the paths used below if a different workspace should be used instead).
+The following sections show how to build the Jazzy version of the Agent in a dedicated workspace (adapt the paths used below if a different workspace should be used instead).
 
 Note: always make sure to use a version of the Agent which corresponds to the version of ROS 2 that is being used.
 For ROS 2 Foxy, checkout the `foxy` branch.
```

Comment:
I suggest removing this `foxy` line.

Reply:
I can do that. I wasn't sure if I should remove references to foxy/galactic in general or leave them in. Should I leave the references in, with the exception of this line?

Reply:
Yeah. IMO, we can say that we support foxy, but we don't want to encourage it in the instructions.

Reply:
I would suggest to keep this, as it's just part of the "if you are still using Foxy, you should be using the Foxy version of the bridge" instructions. It might be deprecated, but we should not make it harder to find the correct workflow for users needing to build the bridge.

There are still quite some people using Foxy and the Foxy version of MotoROS2.